src/components/drivers/photo-viewer.jsx

```diff
@@ -1,17 +1,37 @@
 // Copyright (c) 2017 PlanGrid, Inc.
 
-import React, { Component } from 'react';
+import React, { PureComponent } from 'react';
 
 import 'styles/photo-viewer.scss';
 
-export default class PhotoViewer extends Component {
+export default class PhotoViewer extends PureComponent {
 
 
   componentDidMount() {
     this.insertImage();
+    this.binded = this.onResize.bind(this)
+    window.addEventListener("resize",this.binded)
   }
 
-  componentDidUpdate() {
+	componentWillUnmount() {
+		window.removeEventListener("resize",this.binded)
+  }
+
+	onResize(){
+    console.log("resize")
+		const { originalWidth, originalHeight } = this.props;
+		const imageDimensions = this.getImageDimensions.call(this, originalWidth, originalHeight);
```

Comment:
why not `const imageDimensions = this.getImageDimensions(originalWidth, originalHeight);`?